Build/Props/CodeJam.Targeting.props

```diff
@@ -1,25 +1,30 @@
 ﻿<Project>
 	<!-- BASEDON https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/cross-platform-targeting -->
+
 	<!--
 		Explanation:
-		* We DO target netstandard2.0 as most convenient target
+		* We DO most development and testing on netcoreapp3.0 target
+		* We DO include netstandard2.0 as most convenient target
 		* We DO include net461 as recommended by targeting guidelines
-		* We DO support all real-use targets (net472, netstandard2.1, netcoreapp3.0).
-		* We DO support legacy targets (net45 and earlier, netstandard1.x, netcoreapp1.x) with help of Theraot.Core package
+		* We DO support all actual targets (net5.0, netstandard2.1, net472).
+		* We DO support some legacy targets (net45 and earlier, netstandard1.x, netcoreapp1.x) with help of Theraot.Core package
 		* For tests we use Net Core v1.1 and v2.1 as earlier versions are not included into AppVeyor build images
 	-->
-	<PropertyGroup>
-		<CopyMeMinimalTargetFrameworks>net461;netstandard2.0</CopyMeMinimalTargetFrameworks>
-		<CopyMeMinimalTestTargetFrameworks>net461;netcoreapp2.0</CopyMeMinimalTestTargetFrameworks>
-		<CopyMeTargetFrameworks>net472;net461;net45;net40;net35;net20;netstandard2.1;netstandard2.0;netstandard1.5;netstandard1.3;netcoreapp3.0;netcoreapp2.0;netcoreapp1.0</CopyMeTargetFrameworks>
-		<CopyMeTestTargetFrameworks>net48;net472;net471;net47;net461;net45;net40;net35;net20;netcoreapp3.0;netcoreapp2.1;netcoreapp1.1</CopyMeTestTargetFrameworks>
-	</PropertyGroup>
+	<!-- Generated by CodeJam.TargetingTests.GenerateTargetingProps -->
 
-	<PropertyGroup Condition=" '$(TargetFramework)' == 'net461' ">
+	<!-- We DO most development and testing on netcoreapp3.0 target -->
+	<PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
 		<DefineConstants>$(DefineConstants);DEFAULT_PLATFORM</DefineConstants>
 	</PropertyGroup>
 
-	<!-- Generated by CodeJam.TargetingTests.EmitTargetingConstants -->
+	<!-- Templates for <TargetFrameworks/> project property -->
+	<PropertyGroup>
+		<CopyMeTargetFrameworks>netcoreapp3.0;net5.0;netcoreapp2.0;netcoreapp1.0;netstandard2.1;netstandard2.0;netstandard1.5;netstandard1.3;net472;net461;net45;net40;net35;net20</CopyMeTargetFrameworks>
+		<CopyMeTestTargetFrameworks>netcoreapp3.0;net5.0;netcoreapp2.1;netcoreapp1.1;net472;net461;net45;net40;net35;net20</CopyMeTestTargetFrameworks>
```

Comment:
```suggestion
		<CopyMeTestTargetFrameworks>net5.0;netcoreapp3.0;netcoreapp2.1;netcoreapp1.1;net472;net461;net45;net40;net35;net20</CopyMeTestTargetFrameworks>
```

Reply:
VS does select the first moniker in list as a default target framework, so netcoreapp3.0 should be first one